nautobot/extras/templates/extras/plugin_detail.html

```diff
@@ -146,6 +146,27 @@ <h1>{% block title %}{{ object.verbose_name | bettertitle }}{% endblock %}</h1>
                                     {% endif %}
                                 </td>
                             </tr>
+                            <tr>
+                                <td>Filters</td>
+                                <td>
+                                    {% if features.filter_extensions %}
+                                        <b>FilterSet Fields</b>
```

Comment:
Consider adding logic to omit this heading if `features.filter_extensions.filterset_fields` is empty. Similar suggestion on the `<b>FilterForm Fields</b>` heading below.